blocks/footer/footer.css

```diff
@@ -1,14 +1,267 @@
-footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
+footer .footer-wrapper {
+  background: #fcfcfc;
 }
 
-footer .footer {
-  max-width: 1200px;
+footer .section.footer-top-note {
+  padding-top: 0;
+  padding-bottom: 1.25rem;
+}
+
+.footer-links .section-container{
+  padding-left: 3.438rem;
+  padding-right: 3.438rem;
+  border-bottom: 0.0625rem solid #979797;
```

Comment:
Please use constant from the style.css for this color, if its not there we can try to add it. Same for other color usages places in this PR

Reply:
done